cg/meta/transfer/external_data.py

```diff
@@ -144,43 +144,51 @@ def add_files_to_bundles(
             LOG.info("Adding path %s to bundle %s in housekeeper" % (path, lims_sample_id))
             self.housekeeper_api.add_file(path=path, version_obj=last_version, tags=HK_FASTQ_TAGS)
 
+    def get_failed_samples(self, fastq_paths_to_add: List[Path]) -> List[Path]:
+        failed_sums: List[Path] = []
```

Comment:
I think it is misleading to call this "failed_sums", since it stores Paths. Why not "failed_check_sum_fastq_paths" or at least "failed_sum_paths" since that is what you are storing and not the sums.